source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,276 @@
+===========
+Aggregation
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Aggregation operations process data in your MongoDB collections and return computed results.
+
+A powerful way to think of aggregation is like a car factory. Within the car factory is an assembly line, along which
+are assembly stations with specialized tools to do a specific job, like drills and welders. Raw parts enter the factory,
+which are then transformed and assembled into a finished product.
+
+The **aggregation pipeline** is the assembly line, **aggregation stages** are the assembly stations, and
+**operator expressions** are the specialized tools.
+
+Aggregation and Find Operations Compared
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using ``find`` operations, you can:
+
+- select *what* documents to return
+- select *what* fields to return
+- sort the results
+
+Using ``aggregation`` operations, you can:
+
+- perform all ``find`` operations
+- rename fields
+- calculate fields
+- summarize data
+- group values
+
+Aggregation operations have some :manual:`limitations <core/aggregation-pipeline-limits/>` you must keep in mind:
+
+- Returned documents must not violate the :manual:`BSON Document Size limit </reference/limits/#BSON-Document-Size>`,
+  currently 16 megabytes.
```

Comment:
Thought it was strange to mix title case and normal case within the same term. Also thought "currently" signaled that it's a value that changes often.

```suggestion
- Returned documents must not violate the :manual:`BSON document size limit </reference/limits/#BSON-Document-Size>`
  of 16 megabytes.
```